### PR TITLE
Modify Dockerfile to suppot TLS connection in golang

### PR DESCRIPTION
**Pull request by iv0rish**
## Proposed changes

- Add `update-ca-certificates` process in Klaytn **Dockerfile** to prevent TLS connection error `x509: certificate signed by unknown authority` when using `http.Get` function

## Types of changes

Please put an x in the boxes related to your change.

- [x] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -27,7 +27,8 @@ ARG SRC_DIR
 ARG PKG_DIR
 
 RUN apt update
```
Comment (odg0318) [inline Dockerfile:29]: We can merge the multiple `RUN` blocks like the followings.
```suggestion
RUN apt update; \
            apt install -yq musl-dev ca-certificates; \
            update-ca-certificates; \
            mkdir -p $PKG_DIR/conf $PKG_DIR/bin 
```

File: .dockerignore [hunk 2]
```diff
@@ -1,8 +1,3 @@
-**/.git
```
Comment (odg0318) [inline .dockerignore:1]: Multi stage build is used to build Dockerfile so we need not these rules anymore. These rules trigger `.git` directory is not properly copied inside a container.

File: Dockerfile [hunk 3]
```diff
@@ -17,18 +17,18 @@ ENV KLAYTN_STATIC_LINK=$KLAYTN_STATIC_LINK
 ARG KLAYTN_DISABLE_SYMBOL=0
 ENV KLAYTN_DISABLE_SYMBOL=$KLAYTN_DISABLE_SYMBOL
 
-RUN git init
-ADD . $SRC_DIR
-RUN git init
-RUN cd $SRC_DIR && make all
+WORKDIR $SRC_DIR
```
Comment (odg0318) [inline Dockerfile:20]: `git init` is not required here.

File: Dockerfile [hunk 4]
```diff
@@ -17,18 +17,18 @@ ENV KLAYTN_STATIC_LINK=$KLAYTN_STATIC_LINK
...
 FROM --platform=linux/amd64 ubuntu:20.04
 ARG SRC_DIR
 ARG PKG_DIR
 
-RUN apt update
-RUN apt install -yq musl-dev
-RUN mkdir -p $PKG_DIR/conf $PKG_DIR/bin
+RUN apt update && \
+            apt install -yq musl-dev ca-certificates && \
+            update-ca-certificates && \
```
Comment (odg0318) [inline Dockerfile:30]: CA certificate must be downloaded for TLS connection.

**Comment by iv0rish** [comment]
@aidan-kwon PTAL, Donny and I have tested with `klaytn-deploy` working well.

cc. @henry-will

**Comment by henry-will** [comment]
> @aidan-kwon PTAL, @odg0318 and I have tested with `klaytn-deploy` working well.
> 
> cc. @henry-will

**Comment by iv0rish** [comment]
> > @aidan-kwon PTAL, @odg0318 and I have tested with `klaytn-deploy` working well.
> 
> What about check 1.prepare.sh in `local-klaytn-deploy`?

`local-klaytn-deploy` will also works well, the issue happens on **homi setup**.

**Comment by odg0318** [comment]
> > @aidan-kwon PTAL, @odg0318 and I have tested with `klaytn-deploy` working well.
> 
> What about check 1.prepare.sh in `local-klaytn-deploy`?

Can you confirm whether the issue happens with this branch? I will guide how to do that.

**Review by odg0318 — APPROVED** [review]

**Review by henry-will — APPROVED** [review]

**Review by jeongkyun-oh — APPROVED** [review]

**Pull request merged**